data/json/items/tools.json

```diff
@@ -8270,5 +8270,34 @@
     "ammo": "battery",
     "initial_charges": 100,
     "max_charges": 100
+  },
+  {
+    "result": "washboard",
```

Comment:
I mistakenly deleted that recipe and I mistakenly re-insert it on the wrong json.